projects/wmf_default.yaml

```diff
@@ -141,9 +138,10 @@ paths:
             /mobileapps:
               x-modules:
                 - path: sys/mobileapps.yaml
-                  options: '{{options.mobileapps}}'
+                  options: '{{merge({"response_cache_control": options.purged_cache_control}, 
```

Comment:
This is using the default global setting for mobileapps, and merges it into the other mobileapps settings. The alternative is to define the cache-control setting twice in the config.